source/features/clone-branch.tsx

```diff
@@ -0,0 +1,68 @@
+import gitBranch from 'octicon/git-branch.svg';
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate, {DelegateEvent} from 'delegate-it';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getCleanPathname} from '../libs/utils';
+
+async function cloneBranch(event: DelegateEvent<MouseEvent, HTMLAnchorElement>): Promise<void|false> {
+	const currentTarget = event.delegateTarget;
+	const branchElement = (currentTarget.closest('[data-branch-name]') as HTMLAnchorElement);
+	const newBranchName = prompt('Enter the new branch name');
+	if (!newBranchName) {
+		return false;
+	}
+
+	const spinner = select('.js-loading-spinner', branchElement)!;
+	spinner.hidden = false;
+	currentTarget.hidden = true;
+
+	const [user, repo] = getCleanPathname().split('/');
+	const getBranchInfo = await api.v3(`repos/${user}/${repo}/git/refs/heads/${branchElement.dataset.branchName!}`);
+	try {
```

Comment:
Might as well catch this error too
```suggestion
	try {
		const getBranchInfo = await api.v3(`repos/${user}/${repo}/git/refs/heads/${branchElement.dataset.branchName!}`);
```

Reply:
How would it error? But sure.